.github/workflows/reusable-build-on-macos.yml

```
@@ -45,15 +45,19 @@ jobs:
         run: |
           brew uninstall cmake # conflict with the one used by WasmEdge/llvm/lld
           brew tap WasmEdge/llvm
-          brew install llvm ninja wabt grpc
-          brew install --build-from-source WasmEdge/llvm/lld
+        uses: tecolicom/actions-use-homebrew-tools@v1
+        with:
+          tools: llvm ninja wabt grpc
+      - run: brew install --build-from-source WasmEdge/llvm/lld
       - name: Setup build environment - release
         if: ${{ inputs.release }}
         run: |
```

actionlint (1.7.10)

[error] unexpected key "run" for step to execute action. expected one of "continue-on-error", "env", "id", "if", "name", "timeout-minutes", "uses", "with"

(syntax-check)